shelley-ma/formal-spec/utxo.tex

```diff
@@ -7,7 +7,7 @@ \subsection*{UTxO Helper Functions}
   \emph{Type Definition}
   \begin{equation*}
     \begin{array}{r@{~\in~}l@{\qquad=\qquad}lr}
-      \var{mest} & \MemoryEstimate & \N
+      \var{mest} & \MemoryEstimate & \Integer
```

Comment:
i changed it because auditors pointed this out

Reply:
Yeah, but it was pointed out that this is a difference between spec and code, and I'd say the code is wrong in this case

Reply:
i think we should keep them aligned, but if this will prevent merginig this PR in the near future, feel free to change

Reply:
I also want things to be aligned, but they should be aligned to what should be correct, instead of to something that just happened to end up in the code